src/NHSD.BuyingCatalogue.Ordering.Api/Models/AddressModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace NHSD.BuyingCatalogue.Ordering.Api.Models
+{
+    public sealed class AddressModel

```

Comment:
Class can be `internal`.

Reply:
They need to be public, or in the contructor in the update method it errors. I can either make two different models (update and get) with the get internal and update public, but seen as they have exactly the same content it seems a bit pointless

Reply:
Yes, makes sense – will have been fine as `internal` until you introduced the put.